listener/subgraph_definition/schema.graphql

```diff
@@ -21,4 +21,20 @@ type Withdraw @entity {
     amount: BigInt!
     slot: BigInt!
     slotIndex: Int!
+}
+
+type StandingSellOrderBatch @entity {
+    id: ID!
+    accountId: Int!
+    batchIndex: Int!
+    validFromAuctionId: Int!
+    orders: [OrderDetails!]
+}
+
+type OrderDetails @entity {
```

Comment:
I think so, I guess we can add a None and use the same model.
Once I merge the PR from David, I'll make sure to review this part (I'll leave this unsolved for now)

Reply:
> reuse the existing Order entity

I really hope this is the way we go, because the Auction Settlement Listener is going to have to fetch all orders and work with them. In fact, as I see it, `slot` and `slot_index` will be necessary here just as well...

Reply:
Thanks Ben!

After your nice explanation about what are these slots for standing orders, I changed how the order is "unpacked" from the event. Now it hands to the "`From<bytes> --> Order`" a bit more info, so I can deduct these fields.

The changes are in this commit:
https://github.com/gnosis/dex-services/commit/940943487a0ddff0c9cb6145f8f874222bbb0e6c

I adapted the test for `test_from_log` and it's passing now. 
I know the test `test_to_and_from_entity` it's still failing, cause I don't have finished that part (I still have to figure out how to save entities containing other entities)